sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ManagementChannelTests.java

```diff
@@ -524,6 +525,30 @@ void cancelScheduledMessagesWithEmptyIterable() {
                 .verifyComplete();
     }
 
+    @Test
+    void createRule() {
+        // Arrange, act, assert
+        final String ruleName = "foo-bar";
+        CreateRuleOptions options = new CreateRuleOptions();
+        StepVerifier.create(managementChannel.createRule(ruleName, options))
+            .verifyComplete();
+    }
```

Comment:
I don't think these tests are enough. It shows that the method doesn't throw... we may want to assert things like.. did we call the REquestResponse channel with properties I expected?

Here's an example:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ManagementChannelTests.java#L163-L184

Similarly with the ones below...